src/heretic/sb_bar.c

```diff
@@ -1300,33 +1356,304 @@ static void CheatShowFpsFunc(player_t* player, Cheat_t* cheat)
     }
 }
 
-static void CheatMassacreFunc(player_t * player, Cheat_t * cheat)
```

Comment:
Now the diff is becoming a bit confusing. Is this because you re-arranged the new Crispy-specific cheat codes to come right after the regular ones? If yes, I appreciate that!